app/routes/teams.rb

```diff
@@ -163,20 +164,37 @@ class Teams < Core
       end
 
       # Accept an invitation to join a team.
-      put '/teams/:slug/confirm' do |slug|
+      put '/teams/:slug/accept_invite' do |slug|
         please_login
         only_with_existing_team(slug) do |team|
-          unless team.unconfirmed_members.include?(current_user)
+          unless team.member_invites.include?(current_user)
             flash[:error] = "You don't have a pending invitation to this team."
             redirect "/"
           end
 
-          team.confirm(current_user.username)
+          team.accept_invite(current_user.username)
 
+          flash[:success] = "You are now a member of team #{slug}."
           redirect "/teams/#{slug}"
         end
       end
 
+      # Refuse an invitation to join a team.
+      put '/teams/:slug/refuse_invite' do |slug|
```

Comment:
Or "refuse" instead of "reject". I wasn't reading closely, and don't have a strong opinion either way.